Tests broken again, this time due to `AttributeError`

Tests fail with:
```
______ ERROR collecting tests/integration/interaction/test_wheel_zoom.py _______
tests/integration/integration_tests_plugin.py:45: in pytest_generate_tests
    if metafunc.config.option.driver == "SauceLabs":
E   AttributeError: 'CmdOptions' object has no attribute 'driver'
```
All test runs are affected, because the error happens during test collecting by py.test.

--- a/conda.recipe/meta.yaml
+++ b/conda.recipe/meta.yaml
@@ -44,6 +44,7 @@ requirements:
 
 test:
   requires:
+    - conda ==4.2.15
     # docs
     - pygments
     - sphinx >=1.3.1

--- a/scripts/travis_install
+++ b/scripts/travis_install
@@ -40,7 +40,7 @@ if [ $BUILD_FILE_COUNT != 3 ]; then
    exit 1
 fi
 
-conda install bokeh --use-local --yes
+conda install bokeh conda==4.2.15 --use-local --yes
 
 DEPS_RUN=$(cat <<EOF | python -
 from conda_build.metadata import MetaData
@@ -60,7 +60,7 @@ conda install --yes $DEPS_TEST
 # conda nb extension is installed by default and breaks notebook image diff tests
 python -m nb_conda_kernels.install --disable --prefix=/home/travis/miniconda
 
-conda install --yes nodejs
+conda install --yes nodejs conda ==4.2.15
 echo "node version $(node -v)"
 echo "npm version $(npm -v)"
 

--- a/scripts/travis_install
+++ b/scripts/travis_install
@@ -27,6 +27,7 @@ conda install --yes $DEPS_TRAVIS
 conda config --append channels bokeh
 conda config --append channels conda-forge
 conda config --get channels
+conda config --set auto_update_conda off
 
 CONDA_PY="${TRAVIS_PYTHON_VERSION/./}" conda build --quiet --no-test conda.recipe
 
@@ -40,7 +41,7 @@ if [ $BUILD_FILE_COUNT != 3 ]; then
    exit 1
 fi
 
-conda install bokeh conda==4.2.15 --use-local --yes
+conda install bokeh --use-local --yes
 
 DEPS_RUN=$(cat <<EOF | python -
 from conda_build.metadata import MetaData
@@ -60,7 +61,7 @@ conda install --yes $DEPS_TEST
 # conda nb extension is installed by default and breaks notebook image diff tests
 python -m nb_conda_kernels.install --disable --prefix=/home/travis/miniconda
 
-conda install --yes nodejs conda ==4.2.15
+conda install --yes nodejs
 echo "node version $(node -v)"
 echo "npm version $(npm -v)"
 

